cde-pentaho5/resource/resources/custom/amd-components/Dashboard/component.xml

```diff
@@ -64,6 +65,19 @@
             <Version>1.0</Version>
           </Header>
         </DesignerProperty>
+        <DesignerProperty>
+          <Header>
+            <Name>oneWayMap</Name>
+            <Parent>BaseProperty</Parent>
+            <DefaultValue>"false"</DefaultValue>
+            <Description>One Way Map</Description>
+            <Tooltip>If 'true' parameter propagation will only go in one direction</Tooltip>
```

Comment:
I don't think we need the single quotes surrounding *true*. Checked the tooltips for the other custom components and single quotes seem to only be used for abbreviations.